go.mod

```diff
@@ -200,10 +204,15 @@ replace (
 	github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.7.0
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
+	github.com/linxGnu/grocksdb => github.com/yihuang/grocksdb v1.7.11-0.20221223014353-a4fa14d1759b
+
 	// TODO: remove when gravity update dependencies
 	github.com/peggyjv/gravity-bridge/module/v2 => github.com/crypto-org-chain/gravity-bridge/module/v2 v2.0.1-0.20220815102151-48275db7e1ee
 	github.com/tendermint/tendermint => github.com/tendermint/tendermint v0.34.24-0.20221110131553-ec471ba27efd
 
+	// https://github.com/yihuang/tm-db/tree/rocksdb-v7
+	github.com/tendermint/tm-db => github.com/yihuang/tm-db v0.0.0-20230103092224-3f21822ed03e
```

Comment:
https://github.com/tendermint/tm-db/pull/312
migrate to grocksdb to support rocksdb v7.